Cargo.toml

```diff
@@ -22,9 +27,13 @@ doc = false
 [dependencies]
 async-trait = "0.1.59"
 base16 = "0.2.1"
-casper-types = { version = "3.0.0", features = ["std"] }
-clap = { version = "4", features = ["cargo", "deprecated", "wrap_help"] }
+casper-types = { version = "3.0.0", git = "https://github.com/casper-network/casper-node.git", branch = "feat-2.0", features = [
```

Comment:
I don't have a strong preference on how the cargo.toml is formatted. I'm just curious what lad to the changes

Reply:
I needed to compile with std and then to wasm so I needed to remove calls to io/fs and system calls. So with Fraser we tried to see what was the sammlest changes. It was mostly clap and schemars giving issues. Then the rest is adding the feature it self (which is a restricting feature while a feature should actually be additive) and libs `"cdylib", "rlib"`.
Biggest change is features = ["js"]  to getrandom, I am not sure how to workaround that.

```
   Compiling getrandom v0.2.10
error: the wasm*-unknown-unknown targets are not supported by default, you may need to enable the "js" feature. For more information see: https://docs.rs/getrandom/#webassembly-support
   --> /opt/rust/.cargo/registry/src/index.crates.io-6f17d22bba15001f/getrandom-0.2.10/src/lib.rs:285:9
    |
285 | /         compile_error!("the wasm*-unknown-unknown targets are not supported by \
286 | |                         default, you may need to enable the \"js\" feature. \
287 | |                         For more information see: \
288 | |                         https://docs.rs/getrandom/#webassembly-support");
    | |________________________________________________________________________^

error[E0433]: failed to resolve: use of undeclared crate or module `imp`
   --> /opt/rust/.cargo/registry/src/index.crates.io-6f17d22bba15001f/getrandom-0.2.10/src/lib.rs:341:9
    |
341 |         imp::getrandom_inner(dest)?;
    |         ^^^ use of undeclared crate or module `imp`
    
    ```

The formatting changes are my VsCode with probably Better Toml extention that is making formatting on save.

Reply:
I removed "json-schema" that was added back apparently.

Reply:
I included `getrandom/js` only for sdk feature, this should be less invasive for default build.

Reply:
Actually `getrandom` was removed from dev too completely already I missed that so this issue does not exist anymore.